test/helpers.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node: true */
+
+'use strict';
+
+const brAccount = require('bedrock-account');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const uuid = require('uuid/v4');
+
+const api = {};
+module.exports = api;
+
+api.createAccount = email => {
+  const newAccount = {
+    id: 'urn:uuid:' + uuid(),
+    email
+  };
+  return newAccount;
+};
+
+api.createIdentity = userName => {
```

Comment:
I don't think this is used anywhere ... can we remove? Same for anything else unused.